src/styles/block/cards.scss

```diff
@@ -0,0 +1,13 @@
+@import "../utils/variables";
+@import "../utils/mixin";
+@import "../utils/reset";
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  width: 944px;
```

Comment:
there should not be fixed width, it should be calculated by formula like `max-width: ($card-width*$card-per-row)
    +($hor-padding*2)+($gap*$card-per-row)-$gap;`